src/guc.c

```diff
@@ -675,6 +740,17 @@ _guc_init(void)
 							 NULL,
 							 NULL);
 
+	DefineCustomBoolVariable(MAKE_EXTOPTION("enable_columnarscan"),
```

Comment:
How do TAM tables work when the columnar scan is disabled? Trying to understand user experience, i.e. what does it mean to use the TAM but disable the columnar scan. Maybe the ability to choose the access method is enough?

Reply:
So the columnar scan node is technically not required, and we can use the normal Postgres scan over the custom table access method? Might be good to clarify this in the comment, i.e. "otherwise, the normal postgres Scan node is used over the table with the custom access method."

Reply:
done.